src/keycloak/chart/templates/statefulset.yaml

```diff
@@ -112,7 +112,13 @@ spec:
             - "--spi-authentication-sessions--map--auth-sessions-limit={{ .Values.realmConfig.maxInFlightLoginsPerUser | default 300 }}"
             - "--proxy-headers=xforwarded"
             - "--http-enabled=true"
+            - "--hostname-backchannel-dynamic=true"
+            {{- if .Values.useFixedHostname }}
+            - "--hostname=https://sso.{{ .Values.domain }}"
```

Comment:
Note there's no `--hostname-strict=true` here as it's fully redundant. See https://github.com/keycloak/keycloak/blob/060847a80a36ff597d9167f47e4eaa94042f2ec0/services/src/main/java/org/keycloak/url/HostnameV2ProviderFactory.java#L50-L57